avatax_connector/__manifest__.py

```diff
@@ -1,14 +1,17 @@
-# -*- coding: utf-8 -*-
 {
     "name": "Avalara Avatax Connector",
-    'version': '12.0.1.0.0',
-    "author": "Fabrice Henrion, Sodexis Inc<dev@sodexis.com>",
-    "website": "http://www.sodexis.com",
+    "version": "12.0.2.0.0",
+    "author": "Fabrice Henrion, Sodexis Inc<dev@sodexis.com>"
```

Comment:
@dreispt 
could you please change it from "Sodexis Inc<dev@sodexis.com>" to "Sodexis"

Reply:
```suggestion
    "author": "Fabrice Henrion, Sodexis"
```